client/modules/totalScore.jsx

```diff
@@ -8,11 +8,11 @@ class TotalScore extends React.Component {
 
   render() {
     return (
-    <div>
+    <div className={styles.reviews}>
```

Comment:
Is this how you link your CSS class rules to a certain component using CSS modules? Pretty cool stuff!